src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/BlobInfo.java

```diff
@@ -11,12 +13,12 @@
     public final String fileName;
 
     @JsonProperty("created_at")
-    public final String createdAt;
+    public final Instant createdAt;
```

Comment:
Should also have
`@JsonSerialize(using = InstantSerializer.class)`
annotation

Reply:
done